sklearn/utils/graph.py

```diff
@@ -70,12 +70,6 @@ def single_source_shortest_path_length(graph, source, cutoff=None):
     return seen  # return all path lengths as dictionary
 
 
-if hasattr(sparse, 'connected_components'):
```

Comment:
We maybe need to deprecate the presence of `connected_components` in this module.